### DISCUSSION TITLE
Use ligatures from fallback font

### DISCUSSION THREAD

**Discussion by RayJameson** [wezterm/wezterm#4692] [Ideas]
The title. I like Meslo font, however, it doesn't support ligatures and I have to manually patch the font using ToxicFrog's Ligaturizer every Nerd Font update. So it would be awesome to use ligatures from Fira Code, or Jet Brains Mono. Here is how I see it in the config:

```lua
config.font = wezterm.font_with_fallback {
  { family = "MesloLGSDZ Nerd Font Mono", harfbuzz_features = { "calt=0", "clig=0", "liga=0" } },
  { family = "JetBrains Mono" },
}
```

**Comment by wez** [comment]
Ligatures are part of a font, and need careful consideration on the part of the font author. It is a very complex area and out of scope for wezterm to provide a way to dynamically patch a font through configuration, so this will never be part of wezterm.

**Reply by RayJameson** [reply]
thank you for answer!
